tests/requirements.yml

```diff
@@ -1,14 +1,11 @@
-- src: https://github.com/idealista/mysql-role.git
-  scm: git
+- src: idealista.mysql-role
   version: 1.0.0
   name: mysql
 
-- src: https://github.com/idealista/wildfly-role.git
-  scm: git
+- src: idealista.wildfly-role
   version: 1.1.2
   name: wildfly
 
-- src: https://github.com/idealista/java-role.git
-  scm: git
+- src: idealista.java-role
   version: 1.0.0
```

Comment:
Not really relevant because it would be for testing, but Java and MySQL roles are in the 1.2.0 version

Reply:
I tested to upgrade both versions and didn't works (the server returns '404 - Not Found' in both situations). 

Should be done in https://github.com/idealista/zanata-role/issues/9